Backend/api/src/main.py

```diff
@@ -45,9 +46,48 @@ class UserCollection(BaseModel):
 @app.get("/users/", response_description="List of users", response_model=UserCollection,
     response_model_by_alias=False, )
 async def list_users():
+    #Get list of all user info
```

Comment:
In order to maintain clean code please put a space between your "#" and the line of code.